features/networking/sctp.feature

```diff
@@ -33,23 +32,257 @@ Feature: SCTP related scenarios
    
     # sctpserver pod start to wait for sctp traffic
     When I run the :exec background client command with:
-      | pod              | sctpserver          |
-      | namespace        | <%= project.name %> |
-      | oc_opts_end      |                     |
-      | exec_command     | sh                  |
-      | exec_command_arg | -c                  |
-      | exec_command_arg | nc -l 30102 --sctp  |     
+      | pod              | sctpserver                                |
+      | namespace        | <%= project.name %>                       |
+      | oc_opts_end      |                                           |
+      | exec_command     | sh                                        |
+      | exec_command_arg | -c                                        |
+      | exec_command_arg | /usr/bin/nc -l 30102 --sctp -k -m 5 -w 60 |
     Then the step should succeed
 
     # sctpclient pod start to send sctp traffic
+    And I wait up to 60 seconds for the steps to pass:
+    """"
     When I run the :exec client command with:
-      | pod              | sctpclient                                                       |
-      | namespace        | <%= project.name %>                                              |
-      | oc_opts_end      |                                                                  |
-      | exec_command     | sh                                                               |
-      | exec_command_arg | -c                                                               |
-      | exec_command_arg | echo test-openshift \| nc -v <%= cb.serverpod_ip %> 30102 --sctp | 
+      | pod              | sctpclient                                                               |
+      | namespace        | <%= project.name %>                                                      |
+      | oc_opts_end      |                                                                          |
+      | exec_command     | sh                                                                       |
+      | exec_command_arg | -c                                                                       |
+      | exec_command_arg | /usr/bin/nc -v <%= cb.serverpod_ip %> 30102 --sctp  <<< 'test-openshift' |
```

Comment:
`<<<` is only valid in `bash` not `sh`.   The fact that `sh` is symlinked to `bash` should be not always be assumed as on Debian-based systems `sh` is symlinked to `dash`.   

If we are going to use bash-isms we should be clear and use `bash -c` specifically.

We should probably always use `bash -c`, if we need to use a shell, to ensure we have all the cool bash features.

Reply:
also it looks like `<<<` here-strings are implemented with tmp files in case that matters vs. pipes.

```shell
$ ls -l /proc/self/fd/ <<< "TEST"
total 0
lr-x------. 1 rbrattai rbrattai 64 Aug 25 00:01 0 -> '/tmp/sh-thd.H2fCpL (deleted)'
lrwx------. 1 rbrattai rbrattai 64 Aug 25 00:01 1 -> /dev/pts/14
lrwx------. 1 rbrattai rbrattai 64 Aug 25 00:01 2 -> /dev/pts/14
```
https://askubuntu.com/a/678919

Reply:
Fixed by using "echo test-openshift \| /usr/bin/nc -v <%= cb.serverpod_ip %> 30102 --sctp", thanks!